template.yaml

```diff
@@ -2,11 +2,25 @@ kind: Template
 apiVersion: v1
 template: spark
 metadata:
-  name: spark
+  name: apache-spark
+  creationTimestamp: null
+  annotations:
+    description: Apache Spark with 3 worker nodes.
+    iconClass: icon-nodejs
+    tags: quickstart,Apache,Spark,apachespark
 labels:
   app: sparkcluster
 objects:
-
+- apiVersion: v1
+  kind: PersistentVolumeClaim
```

Comment:
@elmiko, @tmckay please chime in if I'm off-base on this one.  I can appreciate the desire to attach a persistent volume, but I'm not sure that this template is where we want to do that.  In general, we are using the openshift-spark image to spawn spark clusters, which may or may not require a persistent volume.  In a perfect world, we could conditionally include or exclude options such as this, but that is not possible with templates.  Maybe a separate template that includes a persistent volume would be useful.